lib/process.js

```diff
@@ -158,6 +158,7 @@ module.exports = (data, cb) => {
 	const config = data.config;
 	const plugins = data.plugins;
 	const url = data.url;
+	const action = data.action | '';
```

Comment:
1. Looks like we are missing a |
2. Do we really need to put a default value ?
3. Please actually use the const, instead of data.action